RVFitter/RVFitter_comparison.py

```diff
@@ -236,16 +238,21 @@ def compare_fit_results_1D(self, variable, fig_and_ax=None):
         handles, labels = axes[-1].get_legend_handles_labels()
         if variable == 'cen':
             axes[-1].set_xlabel('Velocity (km/s)')
-            filename = "compare_results_velocity.png"
+            if suffix == None:
+                filename = "compare_results_velocity.png"
+            else:
+                print(suffix)
+                filename = suffix + "_compare_results_velocity.png"
         else:
             axes[-1].set_xlabel(variable)
             filename = "compare_results_{variable}.png"
 
         fig.legend(handles, labels, ncol=2, loc='lower center')
-
         this_filename = os.path.join(self.output_folder, filename)
+
         fig.savefig(this_filename)
         print(this_filename, "saved.")
+        plt.show()
```

Comment:
Please add an option to make the `plt.show()` optional.

Reply:
why?

Reply:
If one wants to get these plots on a larger scale it would be very annoying to close the window each time.